jdaviz/data/linelists/UV_linelist_vacuum.csv

```diff
@@ -0,0 +1,132 @@
+Line Name,Rest Value
+C I,1157.19
+N IV,1168.6
+C IV,1168.87
+C IV,1168.99
+N IV,1169.06
+N IV,1169.48
+C III,1175.53
+N III,1183.03
+N III,1184.54
+Mn II,1199.39
+N I,1199.55
+N I,1200.22
+N I,1200.71
+Si III,1206.5
+H I,1215.67
+N V,1238.82
+N V,1242.8
+C III,1247.38
+S II,1250.58
+S II,1253.81
+Si II,1260.42
+C I,1277.24
+Si III,1294.54
+C III,1296.33
+Si III,1296.74
+Si III,1298.93
+O I,1302.17
+Si II,1304.37
+Ni II,1317.22
+C II,1323.93
+N III,1324.35
+C I,1328.83
+C II,1334.53
+C II*,1335.71
+O IV,1341.64
+Fe V,1345.61
+Fe V,1362.86
+Fe V,1363.08
+Fe V,1364.82
+O V,1371.3
+Si IV,1393.76
+Si IV,1402.77
+Si III,1417.24
+C III,1426.45
+Fe V,1427.27
+C III,1427.85
+Fe V,1430.44
+Ni II,1454.84
+Co II,1466.21
+Ni II,1467.26
+Ni II,1467.76
+C III,1478
+N IV],1486.5
+S V,1501.76
+Si II,1526.71
+Fe IV,1530.04
+Si II*,1533.43
+C IV,1548.19
+C IV,1550.77
+C I,1560.31
+C I*,1560.68
+Fe II,1608.45
+Fe II,1611.2
+C III,1620.4
+He II,1640.42
+C I,1656.93
+C I*,1657.38
+C I*,1657.91
+O III],1660.81
+Fe V,1662.32
+O III],1666.15
+Al II,1670.79
+Ni II,1709.6
+Fe IV,1717.9
+N IV,1718.55
+Ni II,1741.55
+Ni II,1751.91
+Si II,1808.01
+Si I,1845.52
+Al III,1854.72
+Al III,1862.79
+Si III],1883
+Si III],1892.03
+Fe II,1901.77
+C III],1906.68
+C III],1908.73
+Fe III,1922.79
+Fe III,1923.01
+Fe III,1923.88
+Fe III,1924.53
+Fe III,1926.04
+Fe III,1926.3
+Fe III,1930.39
+Fe III,1931.51
+Fe III,1932.82
+Fe III,1949.67
+Fe III,1950.33
+Fe III,1951.01
+Fe III,1952.65
+Fe III,1953.33
+Fe III,1954.61
+Fe III,1958.59
+Fe III,1960.32
+Fe III,1962.72
+Fe III,1964.15
+Fe III,1966.41
+Zn II,2026.14
+Cr II,2026.27
+Mg I,2026.48
+Cr II,2056.25
+N II],2140

```

Comment:
Missing `[`?
```suggestion
[N II],2140
```

Reply:
No, "[" is not missing. N II] is the correct entry.

Reply:
Huh, okay. Then you can ignore my other comment. Thanks for the clarification!

Reply:
May be include the reference to this Table somewhere so that people are not confused: https://iopscience.iop.org/article/10.1088/0004-6256/141/2/37/pdf

Reply:
Table 1 in the above reference.